src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/WebResource.java

```diff
@@ -60,7 +63,7 @@
     @ApiModelProperty(position = 3, required = true)
     @NotNull
     @JsonProperty
-    @Column(nullable = false, columnDefinition = "TEXT")
+    @Column(nullable = false, columnDefinition = "VARCHAR")
```

Comment:
Why do we need to set the type here, specially if it's going to be VARCHAR (which I suppose is the default).